private/pkg/git/git.go

```diff
@@ -326,3 +375,20 @@ func OpenRepositoryWithDefaultBranch(name string) OpenRepositoryOption {
 		return nil
 	}
 }
+
+const (
```

Comment:
consts should always be at the top of the file, per the style guide

Reply:
Even if not exported? I'll move them.